velox/docs/functions/spark/aggregate.rst

```diff
@@ -81,4 +81,11 @@ General Aggregate Functions
     Returns the value of `x` associated with the minimum value of `y`.
     Note: Spark provides a non-strictly comparator which is less than or equals to.
     Eg. SELECT min_by(x, y) FROM VALUES (('a', 10)), (('b', 10)), (('c', 50)) AS tab(x, y);
-        > b
\ No newline at end of file
+        > b
+
+.. spark:function:: sum(x) -> x
+
+    Returns the sum of `x`.
```

Comment:
Would you document a list of supported types for x?

```
Supported types are: BIGINT,...
```

See example in : https://facebookincubator.github.io/velox/functions/presto/aggregate.html#bitwise-aggregate-functions